tds-itemselection-aironline/src/main/java/tds/itemselection/algorithms/AdaptiveSelector2013.java

```diff
@@ -28,10 +29,9 @@
 import tds.itemselection.impl.blueprint.ActualInfoComputation;
 import tds.itemselection.impl.blueprint.Blueprint;
 import tds.itemselection.impl.bpmatchcomputation.BPMatchByItemWithIterativeGroupItemSelection;
+import tds.itemselection.impl.item.CsetItem;
 import tds.itemselection.impl.item.PruningStrategySmarter;
-import tds.itemselection.impl.sets.Cset1;
-import tds.itemselection.impl.sets.Cset1Factory2013;
-import tds.itemselection.impl.sets.CsetGroup;
+import tds.itemselection.impl.sets.*;
```

Comment:
Let's not use the `*` imports.  You can change this in IntelliJ under `Settings` -> `Code Style` -> `Imports` and setting the `use *` option to 999 so IntelliJ will never do the splat.